.github/workflows/conformance-aks.yaml

```diff
@@ -56,10 +56,10 @@ concurrency:
     ${{
       (github.event_name == 'schedule' && github.sha) ||
       (github.event_name == 'issue_comment' && (
-        startsWith(github.event.comment.body, 'ci-aks') ||
-        startsWith(github.event.comment.body, '/ci-aks') ||
-        startsWith(github.event.comment.body, 'test-me-please') ||
-        startsWith(github.event.comment.body, '/test')
+        (startsWith(github.event.comment.body, 'ci-aks') && !startsWith(github.event.comment.body, 'ci-aks-')) ||
+        (startsWith(github.event.comment.body, '/ci-aks') && !startsWith(github.event.comment.body, '/ci-aks-')) ||
+        (startsWith(github.event.comment.body, 'test-me-please') && !startsWith(github.event.comment.body, 'test-me-please-')) ||
```

Comment:
Is this required for `test-me-please`? We don't have `test-me-please-1.10`, do we?

Reply:
Not required, just added via grep replace. Decided to leave it because it has no side-effects and will be removed anyway as this is only for the transition period.